Firestore/core/src/nanopb/message.h

```diff
@@ -243,6 +243,11 @@ class SharedMessage {
   std::shared_ptr<Message<T>> message_;
 };
 
+template <typename T>
+SharedMessage<T> MakeSharedMessage(const T& proto) {
+  return SharedMessage<T>(std::move(proto));
```

Comment:
Nit: moving a const reference is extraneous.

Reply:
Removed.